STL_Extension/include/CGAL/assertions.h

```diff
@@ -31,6 +31,27 @@
 
 // #include <CGAL/assertions_behaviour.h> // for backward compatibility
 
+#ifdef NDEBUG
+#  ifndef CGAL_NDEBUG
+#    define CGAL_NDEBUG
+#  endif
+#endif
+
+// The macro `CGAL_DEBUG` allows to force CGAL assertions, even if `NDEBUG`
+// is defined,
+#ifdef CGAL_DEBUG
```

Comment:
No. I have now added the document this morning, in https://github.com/CGAL/cgal/pull/1376/commits/5950bc45d11c8df3702cd6a7b3e03b850a72ea61.